src/developer-quickstart.md

```diff
@@ -0,0 +1,441 @@
+
+```sway
+contract;
+
+storage {
+    counter: u64 = 0,
+}
+
+abi Counter {
+    #[storage(read, write)]
+    fn increment();
+
+    #[storage(read)]
+    fn count() -> u64;
+}
+
+impl Counter for Contract {
+    #[storage(read)]
+    fn count() -> u64 {
+      return storage.counter;
+    }
+    #[storage(read, write)]
+    fn increment(){
+        storage.counter = storage.counter + 1;
+    }
+}
+```
+
+### Build the Contract
+
+Build `counter_dapp` by running the following command in your terminal from inside the `counter_dapp` directory:
+
+```sh
+forc build
+```
+
+You should see something like this output:
+
+```console
+Compiled library "core".
+  Compiled library "std".
+  Compiled contract "counter_dapp".
+  Bytecode size is 240 bytes.
+```
+
+## Testing your Contract
+
+In the directory `tests`, navigate to `harness.rs.` Here you'll see there is some boilerplate code to help you start interacting with and testing your contract.
+
+At the bottom of the file, define the body of `can_get_contract_instance`. Here is what your code should look like to verify that the value of the counter did get incremented:
+
+```sway
+#[tokio::test]
+async fn can_get_contract_id() {
+    let (instance, _id) = get_contract_instance().await;
+    // Now you have an instance of your contract you can use to test each function
+    instance.increment().call().await.unwrap();
+    let result = instance.count().call().await.unwrap();
+    assert!(result.value > 0); 
+}
+```
+
+Run the following command in the terminal: `forc test`.
+
+You'll see something like this as your output:
+
+```console
+ Compiled library "core".
+  Compiled library "std".
+  Compiled contract "counter_dapp".
+  Bytecode size is 208 bytes.
+  Compiling counter+dapp v0.1.0 (</path/to/counter_dapp>)
+  Finished test [unoptimized + debuginfo] target(s) in 11.71s
+  Running tests/harness.rs (target/debug/deps/integration_tests-6a600a6a87f48edb)
+
+running 1 test
+test can_get_contract_id ... ok
+
+test result: ok. 1 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.24s
+```
+
+## Deploy the Contract
+
+It's now time to deploy the contract and call it on a Fuel node. We will show how to do this using `forc` from the command line, but you can also do it using the [Rust SDK](https://github.com/FuelLabs/fuels-rs#deploying-a-sway-contract) or the [TypeScript SDK](https://github.com/FuelLabs/fuels-ts/#deploying-contracts).
+
+We'll be deploying to the Fuel testnet. In order to deploy a contract, you need to have a wallet to sign the transaction. You can optionally pass the flag `--unsigned` to deploy a contract without a signature.
+
+By default fuel-core runs without UTXO validation, which means you can run unsigned transactions. This allows you to send invalid inputs to emulate different conditions.
+
+### Install the Wallet CLI
+
+Follow [these steps to set up a wallet to sign transactions](https://github.com/FuelLabs/forc-wallet#forc-wallet).
```

Comment:
I think it would say to follow all steps and create one address. Maybe it would be great to have it inside the guide as the installation steps don’t have it.

Reply:
the link does have instructions for creating an account, but I updated the text to explicitly say "follow these steps to create a wallet and create an account"

I don't want to manually add the steps because as we make updates, it'll create more places where things may become out of date.

<img width="950" alt="Screen Shot 2022-09-06 at 5 34 01 PM" src="https://user-images.githubusercontent.com/15346823/188763456-055724d7-68b7-4c94-8473-8cfe2d93e5f4.png">